src/services/web3/web3.plugin.ts

```diff
@@ -0,0 +1,148 @@
+import { MetamaskConnector } from './connectors/metamask/metamask.connector';
+import { Connector } from './connectors/connector';
+import { computed, reactive, ref, Ref, toRefs } from 'vue';
+import { WalletConnectConnector } from './connectors/trustwallet/walletconnect.connector';
+import { getAddress } from '@ethersproject/address';
+import { lsGet, lsRemove, lsSet } from '@/lib/utils';
+import { Web3Provider } from '@ethersproject/providers';
+import { WalletLinkConnector } from './connectors/walletlink/walletlink.connector';
+
+export type Wallet = 'metamask' | 'walletconnect' | 'walletlink';
+export const SupportedWallets = [
+  'metamask',
+  'walletconnect',
+  'walletlink'
+] as Wallet[];
+export const WalletNameDictionary: Record<Wallet, string> = {
```

Comment:
What do you think about using 'map' terminology instead of dictionary throughout the app? Dictionary is such a long word! 😅 
```suggestion
export const WalletNameMap: Record<Wallet, string> = {
```

Reply:
Done deal